### ISSUE TITLE
JS TypeError: Safari and IE11: Escape doesn't move focus back from the toolbar to the block content

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#6165]
Noticed while investigating on #6118 / #6163.

When using the keyboard and moving focus from the block content to the toolbar using `Alt+F10`, then pressing Escape is supposed to move focus back from the toolbar to the block content.

This works perfectly on Chrome, Firefox, and Edge. It doesn't work on Safari and IE11.

I've quickly seen there's a check for an actual selection and focus is moved back only if there's a selection. I'd suspect the method to get the selection doesn't work with IE11 (not surprisingly) and Safari.

**Comment by afercia** [comment]
This is what the selection looks like in IE11 (please ignore the other warnings):

![screenshot 109](https://user-images.githubusercontent.com/1682452/38731175-a0437898-3f19-11e8-9139-105ad2495513.png)

Seems there's a selection, but focus is not moved back to the block content.

**Comment by afercia** [comment]
Still a problem in current master 3.8.0-rc.1. Just re-tested in Safari 11.1.2 and getting:

`TypeError: null is not an object (evaluating 'focusElement.nodeType')`

**Comment by afercia** [comment]
Note: this doesn't happen on the TinyMCE demo https://www.tiny.cloud but still happens on current Gutenberg master 4.0. It appears to be something specific to the selection methods used in Gutenberg.

**Comment by aduth** [comment]
Fix proposed at #10529

**Comment by afercia** [comment]
Noting this is still a problem and the JS error still happens. This issue was opened 11 months ago. The PR exploring a solution is #10699 and it's now marked as stale. Some higher priority on this would be greatly appreciated.

**Comment by afercia** [comment]
Also related to pressing Escape from the toolbar: #11774

**Comment by youknowriad** [comment]
I believe this behavior was removed with the navigation mode. So closing this issue.
